crowd-server/FatemeNikkhah/myproject/food/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+
+
+# Create your models here.
+class Question(models.Model):
+    question = models.TextField()
+    image_url =models.TextField()
+    language = models.CharField(max_length=15)
+    question_id = models.PositiveIntegerField(primary_key=True)
+
+
+class Response(models.Model):

```

Comment:
It is maybe foodFact table but:
- There is no relation between foodFact and food questions.